mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
 Resource Dataset Alias,"URL or persistent identifier (DOI, Synapse ID) for any dataset(s) that is intended to be used with the item. Multiple values should be provided as a comma separated list.",,,FALSE,,,,MC2/dataset,list like
 Resource Tool Link,"URL or persistent identifier (DOI, Synapse ID) for any software or tool that is intended to be used with the item. Multiple values should be provided as a comma separated list.",,,FALSE,,,,MC2/tool,list like
 EducationalResource_id,A unique primary key that enables record updates using schematic.,,,TRUE,,,,,unique
+Biospecimen,Metadata attributes for describing a tumor sample,,"Component, Individual ID, Biospecimen ID, Sex, Age at Diagnosis, Biospecimen Tumor Type, Tumor Subtype, Tumor Grade, Metastasis at Diagnosis, Last Known Disease Status, Preservation Method, Site of Resection or Biopsy, Tissue or Organ of Origin, Timepoint Label, Health Care Site",FALSE,,,,,
+Individual ID,ID associated with a study participant,,,TRUE,,,,HTAN,
+Biospecimen ID,ID associated with a biosample,,,TRUE,,,,HTAN,
+Sex,Biological sex associated with the specimen donor,,,FALSE,,,,,
+Age at Diagnosis,Age at the time of diagnosis expressed in number of days since birth.,,,FALSE,,,,HTAN,
+Biospecimen Tumor Type,"The type of tumor associated with the specimen. Multiple values permitted, comma separated.",,,FALSE,,,,MC2,
+Tumor Subtype,"The subtype related to the scientific determination and investigation, analysis and recognition of the presence and nature of disease, condition, or injury from expressed signs and symptoms of tissue growth resulting from uncontrolled cell proliferation.",,,FALSE,,,,,
+Tumor Grade,"Numeric value to express the degree of abnormality of cancer cells, a measure of differentiation and aggressiveness.",,,FALSE,,,,HTAN,
+Metastasis at Diagnosis,The text term used to describe the extent of metastatic disease present at diagnosis.,,,FALSE,,,,HTAN,
+Last Known Disease Status,Text term that describes the last known state or condition of an individual's neoplasm.,,,FALSE,,,,HTAN,
+Preservation Method,Text term that represents the method used to preserve the sample.,,,FALSE,,,,HTAN,
+Site of Resection or Biopsy,"The text term used to describe the anatomic site of the resection or biopsy of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).",,,FALSE,,,,HTAN,
+Tissue or Organ of Origin,"The text term used to describe the anatomic site of origin, of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).",,,FALSE,,,,HTAN,
```

Comment:
Added this to address Tumor Location feature

Reply:
I agree, there is some ambiguity here, though this is how the term is listed and defined within the HTAN model. "Tumor Origin Site" may be a better fit, given the set of valid values available in the MC2 data model, which I expect we will be using for "Site of Resection or Biopsy" as well: https://github.com/mc2-center/data-models/blob/main/modules/shared/tissue.csv